### PR TITLE
Adds detection for Evoo, Gtel, INSYS, Selecline, Turkcell, Türk Telekom, Venturer brands and improves detection for AT&T, Blu, Cloud, Crosscall, Haier, Logic, Maxwest, STG Telecom, T-Mobile, Vastking, X-TIGI brands

### LINKED ISSUE #5898: Turk (Operator Telekom) brand not detection
Hi, Do I need to add devices from mobile operators?

user_agent
---
| id | name | created_at | 
| ---: | --- | --- | 
| 663881 | Mozilla/5.0 (Linux; Android 5.1; Turk Telekom TT175 Build/LMY47D) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Mobile Safari/537.36 | 2018-12-25 20:20:13 | 
| 722929 | Mozilla/5.0 (Linux; Android 5.0.1; Telekom Puls Build/LRX21M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36 | 2018-12-25 20:26:07 | 
| 2320177 | Mozilla/5.0 (Linux; Android 5.0.1; Telekom Puls Build/LRX21M) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.91 Safari/537.36 | 2018-12-26 01:14:38 |

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

**Comment by liviuconcioiu** [comment]
https://github.com/matomo-org/device-detector/issues/5898 should be closed after this merge.

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -3032,7 +3040,7 @@ Bitel:
...
+  regex: 'BLU[ _]|(?:blu|Dash)[ _]([^/;)]+)(?: Build|[;)])|(?:Studio[ _](5.0K|5.5|View XL|Mega|C 8\+8|[CGM][ _]HD|[CGX]|SELFIE|Selfie LTE|Touch|M5 Plus|J[1258]|X8 HD|X10)|Advance (4.0 ?[LM]|5.[02](?: HD)?|A4|L[45])|M6 2021|ENERGY (DIAMOND|XL)|Energy X 2|LIFE XL|B110DL|Dash (?:X[2L]|L3)|PURE (?:X[LR]|MINI|View)|Life One X2|G50 (?:Mega|Plus)|G90(?: PRO)?|GRAND (?:M3|Max|5.5 HD|XL LTE|Mini)|R1 (?:HD|PLUS)|R2 (?:LTE|Plus)|Tank Xtreme [45].0|Tank Xtreme Pro|Touchbook M7 Pro|DASH X PLUS|C[456] 2019|BOLD (?:M7|N1)|B130DL|Vivo (?:5 Mini|One Plus|5R|XL[234]| Plus|8L|XI[+]?|GO)|C[56]L(?: 2020)?|C6 2020|G9 PRO|D701|VIEW 1)(?:[);/ ]|$)'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:3043]: It seems that `C5L` conflicts from UA
* Mozilla/5.0 (Linux; U; Android 5.1.1; fa-ir; Neffos C5L Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Mobile Safari/537.36

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -7247,6 +7267,16 @@ Grundig:
     - regex: 'AFTEU011'
       model: 'Vision 6 HD (2019)'
 
+# Gtel (https://gtel.co.zw/)
+Gtel:
+  regex: 'Gtel'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:7272]:
```yml
regex: 'GTel[ _]'
```

* Mozilla/5.0 (Linux; Android 4.4.4; SM-T561 Build/KTU84P) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/33.0.0.0 Safari/537.36 Instagram 27.0.0.11.97 Android (19/4.4.4; 160dpi; 800x1280; samsung; SM-T561; gtel3g; sc8830; ru_RU)

File: regexes/device/mobiles.yml [hunk 3]
```diff
@@ -18529,6 +18570,16 @@ Venso:
     - regex: '(CX-508|CX-551|Reiv 500|Creon F5)'
       model: '$1'
 
+# Venturer (https://venturer.com/)
+Venturer:
+  regex: 'VCT|VCT9B06Q22(?:[);/ ]|$)'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:18575]:
```yml
regex: 'VCT([a-z0-9_-]+)(?:[);/ ]|$)'   
```

File: regexes/device/mobiles.yml [hunk 4]
```diff
@@ -2930,6 +2930,14 @@ Asus:
       device: 'desktop'
       model: 'Chromebook Flip $1'
 
+# Auchan (https://www.auchan.lu/)
+Auchan:
+  regex: 'S4T7IN(?:[);/ ]|$)'
+  device: 'tablet'
+  models:
+    - regex: 'S4T7IN(?:[);/ ]|$)'
+      model: 'Selecline S4T7IN'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:2939]: rename brand Auchan to Selecline

`Selecline` is a trademark of the retailer (Auchan).
`Qilive` is a trademark of the retailer (Auchan).

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

**Review by sanchezzzhak — APPROVED** [review]

File: regexes/device/mobiles.yml [hunk 5]
```diff
@@ -23718,6 +23788,14 @@ Zatec:
     - regex: 'Zpower_Plus'
       model: 'Zpower Plus'
 
+# Selecline (https://www.auchan.lu/)
+Selecline:
+regex: 'S4T7IN(?:[);/ ]|$)'
+device: 'tablet'
+models:
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:23795]: missed space in section?

File: regexes/device/mobiles.yml [hunk 6]
```diff
@@ -22964,6 +23014,22 @@ TurboKids:
   device: 'tablet'
   model: '$1'
 
+# Turkcell (https://www.turkcell.com.tr/)
+Turkcell:
+  regex: 'TURKCELL'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:23019]:
```yml
regex : 'TURKCELL[_ ]'
```
* Mozilla/5.0 (iPhone; CPU iPhone OS 7_1_2 like Mac OS X) AppleWebKit/537.51.2 (KHTML, like Gecko) Mobile/11D257 [FBAN/MessengerForiOS;FBAV/84.0.0.25.71;FBBV/36443639;FBRV/0;FBDV/iPhone3,2;FBMD/iPhone;FBSN/iPhone OS;FBSV/7.1.2;FBSS/2;FBCR/Turkcell;FBID/phone;FB
* Mozilla/5.0 (Android 6.0.1; SM-J700F Build/MMB29K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3317.0 YaaniBrowser/4.3.0.153 (Turkcell-TR) Mobile Safari/537.36

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

File: regexes/device/mobiles.yml (hunk 6 above)
Comment (sanchezzzhak) [reply regexes/device/mobiles.yml]: others models (optional)

"Dalvik/1.6.0 (Linux; U; Android 4.1.2; TURKCELL T40 Build/JZO54K)"
"Dalvik/1.6.0 (Linux; U; Android 4.4.2; TURKCELL T50 Build/KVT49L)"
"Dalvik/2.1.0 (Linux; U; Android 5.0.2; TURKCELL T60 Build/LRX22G)"
"Mozilla/5.0 (Linux; Android 6.0.1; TURKCELL T70) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.74 Mobile Safari/537.36"
"Dalvik/2.1.0 (Linux; U; Android 7.1.1; TURKCELL T80 Build/NMF26F)"
"Mozilla/5.0 (Linux; Android 4.4.2; TURKCELL TURBO T50) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Mobile Safari/537.36"
"Mozilla/5.0 (Linux; Android 5.1; Turkcell_T_Tablet) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36"
"Mozilla/5.0 (Linux; U; Android 2.1-update1; ru-ru; Turkcell T10 Build/ERE27) AppleWebKit/530.17 (KHTML, like Gecko) Version/4.0 Mobile Safari/530.17"
"Mozilla/5.0 (Linux; U; Android 2.3.6; ru-ru; Turkcell T20) Build/HuaweiU8650 AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1"
"Mozilla/5.0 (Linux; U; Android 4.0.4; tr-tr; Turkcell Maxi Plus 5 Build/IMM76D) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30"
"Mozilla/5.0 (Linux; U; Android 4.0.3; tr-tr; TURKCELL MaxiPRO5 Build/HuaweiU8860) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30"

Comment (liviuconcioiu) [reply regexes/device/mobiles.yml]: All of them added!

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**